gcdl-certification-comprehensive-guide.md

```
@@ -156,30 +180,39 @@ Google Cloud は「コンプライアンスレポートマネージャー」を
+障害に対するレジリエンス（回復力）を高めるため、システム設計時には単一障害点（SPOF）を排除し、Cloud Load Balancing の単一のグローバルAnycast IP アドレスを使用して、世界の複数リージョンにトラフィックをインテリジェントに分散するディザスタリカバリ（DR）アーキテクチャを採用する [47](https://cloud.google.com/blog/topics/developers-practitioners/google-cloud-global-external-https-load-balancer-deep-dive)
+[48](https://docs.cloud.google.com/architecture/dr-scenarios-building-blocks)。
 また、組織の要件に応じた Google Cloud Customer Care（サポート）の階層化されたプランの選択も重要である。
 
 > ※ 以下のサポート料金・SLA情報は 2026-04-06 時点の公式ドキュメントに基づく。最新情報は [Google Cloud サポートページ](https://cloud.google.com/support) で確認すること。
 
 | サポート階層 | 月額コストの体系 | SLA (レスポンスタイム) | 主要な特徴と想定ユースケース |
 | --- | --- | --- | --- |
-| Standard Support | $29 またはクラウド支出の 3% のいずれか高い方 [49](https://cloud.google.com/support)[50](https://cloud.google.com/support/docs/standard) | P2 (High Impact) の課題に対し 4 時間以内 | 試験的なプロジェクトや開発・テスト環境など、ミッションクリティカルでないワークロードに最適 |
-| Enhanced Support | $500 またはクラウド支出の階層的割合 (10%〜3%) [49](https://cloud.google.com/support)[51](https://cloud.google.com/support/docs/enhanced) | P1 (Critical Impact) の課題に対し 1 時間以内 | 高速な対応が求められる本番環境向け。サードパーティテクノロジーのサポートや Active Assist API の利用が含まれる |
-| Premium Support | $15,000 またはクラウド支出の階層的割合 (10%〜3%) [49](https://cloud.google.com/support)[52](https://cloud.google.com/support/docs/premium) | P1 (Critical Impact) の課題に対し 15 分以内 | 停止が許されないエンタープライズ規模のミッションクリティカルなワークロード向け。専任TAMが配置され、アーキテクチャガイダンスや運用健全性レビューを提供 |
+| Standard Support |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 4; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)